src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -130,7 +132,7 @@ public void Ctor_ExpectedDefaultPropertyValues()
             }
         }
 
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "MaxRequestContentBufferSize not used on .NET Core due to architecture differences")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot | TargetFrameworkMonikers.NetFramework, "uap: dotnet/corefx #20010, netfx: MaxRequestContentBufferSize not used on .NET Core due to architecture differences")]
```

Comment:
This could be done as ~TargetFrameworkMonikers.Netcoreapp which will disable for all the TFMs different than netcoreapp.